config/file.v1/file.go

```diff
@@ -112,9 +112,6 @@ func (file File) Server() string {
 }
 
 func (file File) Title() string {
-	if file.CLI.Title == "" {
```

Comment:
Have you tested this to make sure that the correct title still appears when a project is uploaded? Or that it doesn't impact any other code paths?

Reply:
this change just makes the function consistent with all the other config variables which we check in the order: cli argument -> yml file -> etc...  Together with the new `func Title()` the end result is unchanged.